src/portable/nuvoton/nuc121/dcd_nuc121.c

```diff
@@ -395,9 +395,11 @@ void USBD_IRQHandler(void)
           xfer->remaining_bytes -= available_bytes;
           xfer->data_ptr += available_bytes;
 
-          /* when the transfer is finished, alert TinyUSB */
+          /* when the transfer is finished, alert TinyUSB; otherwise, accept more data */
           if ( (0 == xfer->remaining_bytes) || (available_bytes < xfer->max_packet_size) )
             dcd_event_xfer_complete(0, ep_addr, available_bytes, XFER_RESULT_SUCCESS, true);

```

Comment:
the `available_bytes` in `dcd_event_xfer_complete()` should be an accumulated value so far, ideally it is the `original len - remaining_bytes`